sentry_sdk/profiler.py

```diff
@@ -587,13 +584,11 @@ def stop(self):
 
     def __enter__(self):
         # type: () -> Profile
-        hub = self.hub or sentry_sdk.Hub.current
-
-        _, scope = hub._stack[-1]
+        scope = sentry_sdk.scope.Scope.get_isolation_scope()
         old_profile = scope.profile
         scope.profile = self
 
-        self._context_manager_state = (hub, scope, old_profile)
+        self._context_manager_state = (None, scope, old_profile)
```

Comment:
Do we need to keep the first part of the tuple? Actually, do we need `self._context_manager_state` at all? Not sure what this is used for, we just set it here and then delete it while `__exit__`ing.

Reply:
done